cluster/points_writer.go

```diff
@@ -233,6 +237,9 @@ func (w *PointsWriter) WritePoints(p *WritePointsRequest) error {
 		}(shardMappings.Shards[shardID], p.Database, p.RetentionPolicy, points)
 	}
 
+	// Fire and forget points to Subscriber
+	go w.Subscriber.WritePoints(p)
```

Comment:
Granted, the system already launches a goroutine for every write to the shards, so there is precedent. The goroutine should be amortized over the size of the points batch.